test/unit/index.js

```diff
@@ -50,7 +52,12 @@ var paths = [
 
 var parallelTestsRunning = {};
 paths.forEach(function (test) {
-	var child = child_process.spawn('node_modules/.bin/_mocha', ['--timeout', (8 * 60 * 1000).toString(), 'test/unit/' + test], {
+	var coverageArguments = ['cover', '--dir', 'test/.coverage-unit', '--include-pid', 'node_modules/.bin/_mocha', 'test/unit/' + test, '--', '--timeout', (8 * 60 * 1000).toString()];
+	if (typeof process.argv[2] == 'undefined' || process.argv[2] != '@slow') {
```

Comment:
This looks really bad. But anyway should be done before running the task - not inside a loop.